astropy/visualization/wcsaxes/coordinate_helpers.py

```diff
@@ -43,8 +44,10 @@
 def wrap_angle_at(values, coord_wrap):
     # On ARM processors, np.mod emits warnings if there are NaN values in the
     # array, although this doesn't seem to happen on other processors.
+    if type(coord_wrap) is not Quantity:
```

Comment:
Since below you directly take a value again, I would go in the other direction, or catch everything in one go with,
`u.Quantity(coord_wrap, u.deg).value` - this will use `u.deg` as a unit for non-quantity, and convert any `Quantity` to degrees.